src/main/java/build/buildfarm/server/WebController.java

```diff
@@ -0,0 +1,291 @@
+// Copyright 2023 The Bazel Authors. All rights reserved.
```

Comment:
I think we should put all UI related code in a separate ui module (controllers, etc) as I think this will end up growing and hopefully host bfadmin as well.

Reply:
moved to controllers